graphql/src/schema/api.rs

```diff
@@ -240,6 +242,27 @@ fn add_block_height_type(schema: &mut Document) {
     schema.definitions.push(def);
 }
 
+/// Adds a global `BlockChangedFilter` type to the schema. The `block` argument
+/// accepts values of this type
+fn add_changed_block_filter_input_type(schema: &mut Document) {
+    let typedef = TypeDefinition::InputObject(InputObjectType {
+        position: Pos::default(),
+        description: None,
+        name: CHANGE_BLOCK_FILTER_NAME.to_string(),
+        directives: vec![],
+        fields: vec![InputValue {
+            position: Pos::default(),
+            description: None,
+            name: "number_gte".to_owned(),
+            value_type: Type::NamedType("Int".to_owned()),
```

Comment:
That should be `number_gte: Int!`, not just `number_gte: Int`, right?

Reply:
Yeah, that's right :) will fix and move to `meta.graphql`.